### PR TITLE
License Added to credits.html

### PR DESCRIPTION
**Pull request by SilentFlame**
#58 part 2 resolved.

Have a look.

### CODE REVIEW

File: credits.html [hunk 1]
```diff
@@ -38,6 +38,8 @@
 						<li><a href="http://codepen.io/elrumordelaluz/pen/XmygWX">Lionel T</a>(Github Edge)</li>
 						<li><a href="https://github.com/niccokunzmann/rotoscopes/">Rotoscopes</a></li>
 					</ul>
+					<h3>License</h3>
+						<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
```
Comment (niccokunzmann) [inline credits.html]: It should be one tab more to the left.
Please create an "images" folder and put the creative commons picture in there. The game should work without internet connection.

**Comment by SilentFlame** [comment]
Images folder added.

**Comment by niccokunzmann** [comment]
@SilentFlame 
1. Please go to #76 and notify the people working on the issue that you did it already, invite them to review your changes.
2. [ ] part 2 is not resolved. The creative commons picture is still loaded from the internet.

If you do multiple issues in one pull-request, in my understanding:
- One issue is delayed because if the others.
- Complexity rises and it is harder to review.
- The commits are mixed and thus the history is harder to read.
- To create a new branch and a new pull-request for #76, you could use `git checkout` and `git cherry-pick` for it. 

I wold accept it in this case because the issues are so small that the delay does not matter. For bigger issues I would get problems as a reviewer.

**Comment by SilentFlame** [comment]
yes I understand the problem. from next time I'll keep these thins in mind. It's always your guidance and help, don't think it as a hurt or something. Regards. :)

**Comment by SilentFlame** [comment]
- creative commons picture added locally.

- flappy.svg again kept out of images folder.
Have a look.

**Comment by niccokunzmann** [comment]
The license part looks good. 

@mukesh14149, if you like to review the changes made by @SilentFlame, I will wait for your OK to merge it. If you would not like to review it, please tell me and I will do the review some time soon.

**Comment by mukesh14149** [comment]
Ya it looks great! you can merge it @niccokunzmann

**Comment by niccokunzmann** [comment]
Thanks @mukesh14149 and @SilentFlame !

**Pull request merged**